### PR TITLE
start at readme update

### PR DESCRIPTION
**Pull request by lwasser**
This begins the process of updating the readme file.
see:
https://github.com/earthlab/earthpy/issues/302

@mbjoseph what items should we include in the readme? it seems to me that there is some information in the contributing doc that was started here. maybe we can just include links to files as needed? thoughts?

### CODE REVIEW

**Comment by mbjoseph** [comment]
@lwasser I added some suggestions in #302 so we can keep the discussion in the issues (plus we might need to spread the overhaul over multiple PRs).

**Comment by lwasser** [comment]
@all-contributors please add @mbjoseph for being cool

**Comment by lwasser** [comment]
@all-contributors please add @mbjoseph for infrastructure coolness

**Comment by lwasser** [comment]
ok giving up on the pipe dream for the time being given the contributors format is HTML right now and i'm not sure that RTD can handle HTML within md and html files. if it can we can use includes however but i just don't know how to do it.

**Comment by lwasser** [comment]
ok @mbjoseph i've played around with a few things. here's the deal

1. the contributor bot is cool BUT... the issue is that it creates a contrib table in HTML format. and that clashes with RTD unfortunately unless there is some way to include html. this is just fine for our readme as markdown can tolerate HTML (our website uses both) but .rst doesn't like it.
2. so let's look at the readme text and such only in this PR. and maybe i'll keep exploring other ways to update the contributors. 

There could be a clever way to create a little script to do it via the GH api. it wouldn't be hard. but i'd rather find something we can just use!!

File: README.md [hunk 1]
```diff
@@ -7,64 +7,104 @@
...
+EarthPy also has an io module that allows users to 
+
+1. Quickly access pre-created datasubsets used in the earth-analytics courses hosted 
+on [www.earthdatascience.org](https://www.earthdatascience.org) 
+2. Download other datasets that they may want to use in their workflows.
+
+## View Example EarthPy Applications in Our Documentation Gallery  
+
+Check out our [vignette gallery](https://earthpy.readthedocs.io/en/latest/gallery_vignettes/index.html) for 
+applied examples of using earthpy in common spatial workflows.
```
Comment (mbjoseph) [inline README.md]:
```suggestion
applied examples of using EarthPy in common spatial workflows. 
```

File: README.md [hunk 2]
```diff
@@ -7,64 +7,104 @@
...
-yet you can still use the `--upgrade` argument.
+To install, use `pip` or `conda-forge`. We encourage you to use `conda-forge` if you are a conda users. 
 
-```bash
-$ pip install --upgrade git+https://github.com/earthlab/earthpy.git
-```
+### Install via Pip
 
-Then import it into python.
+To install earthPy via `pip` use:
```
Comment (mbjoseph) [inline README.md]:
```suggestion
To install EarthPy via `pip` use:
```

**Comment by lwasser** [comment]
ok @mbjoseph i just merged all of your suggested commits!!

**Pull request merged**